scripts/deploy-workload-cluster.sh

```diff
@@ -3,6 +3,18 @@
 TKG_LAB_SCRIPTS="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 source $TKG_LAB_SCRIPTS/set-env.sh
 
+get_cluster_type () {
```

Comment:
I just realized this is the first time we needed to add conditional logic in this script, because we pass the cluster specific info as params.
And I'm assuming you decided to avoid more params and add logic inside this time since this is a bonus lab and those params are optional. Would that be correct?

Reply:
That's right.  For backward compatibility, I chose to keep the script args unchanged.